### ISSUE TITLE
"How to Make a Computer Operating System" book status

### ISSUE THREAD

**Issue by mir3z** [SamyPesse/devos#11]
Are you going to finish up your book (http://samypesse.gitbooks.io/how-to-create-an-operating-system)? It is really great - exactly what I'm looking for but I can see that only first 8 chapters are readable.

**Comment by SuriyaaKudoIsc** [comment]
:+1:

**Comment by professorlevi** [comment]
Yeah, I have also read the book, it looks great. It would be great and helpful for everyone if you could take out some time and complete the book :)
